website/src/user-guide/discovering-symbols.md

```diff
@@ -126,3 +126,16 @@ parameters:
     scanDirectories:
         - phar://%currentWorkingDirectory%/bin/robo.phar
 ```
+
+Disable the symbol discovery error tip
+---------------------------
```

Comment:
Add the "Available in PHPSta X.Y.Z" badge. It usually looks like this:

```
<div class="text-xs inline-block border border-green-600 text-green-600 bg-green-100 rounded px-1 mb-4">Available in PHPStan 2.1.7</div>
```

The next will be 2.1.12.